README.md

```diff
@@ -5,3 +5,20 @@ This repository contains bootstrap and control plane providers to deploy Canonic
 CABPCK (Cluster API bootstrap provider for Canonical Kubernetes) is responsible for generate cloud-init scripts for generate Machines such that they run Kubernetes nodes. This implementation uses [Canonical Kubernetes](https://github.com/canonical/k8s-snap) to deliver Kubernetes.
 
 CACPCK (Cluster API control plane provider for Canonical Kubernetes) is responsible for managing the lifecycle of machines that host the control plane nodes of a Canonical Kubernetes cluster.
+
+## Tests
+
+Run the unittests with
+
+```shell
+make test-unit
+```
+
+For the e2e tests, run:
+
+```shell
+make docker-build-e2e  # run only once to build the images
+make test-e2e
+```
+
+Visit the [tests README](./test/e2e/README.md) for more information.
```

Comment:
Let's keep the readme clean from development instructions. Also, the e2e README.md should be updated, there's outdated references to tilt etc, let's make sure we clean them up and make it a simple straightforward doc that any engineer who works on this can setup their environment, and run the tests